docs/src/extremevalue/generalities.md

```diff
@@ -0,0 +1,103 @@
+# [Extreme Value Copulas](@id Extreme_theory)
+
+*Extreme value copulas* are fundamental in the study of rare and extreme events due to their ability to model dependency in situations of extreme risk. An extreme value copula $C$ has the property that $$C(u_1^t, u_2^t)=(C(u_1,u_2))^t, t > 0,$$
+and can be represented by
+$$C(u_1, u_2)=\exp\{-\ell(\log(u_1),\log(u_2))\}$$
+or
+$$C(u_1,u_2)=\exp\left\{\log(u_1u_2)A\left(\frac{\log(u_1)}{\log(u_1u_2)}\right)\right\}.$$ 
+Here, $\ell(\cdot)$ is the stable tail dependence function and $A$ is the Pickands dependence function where $A: [0,1] \to [1/2, 1]$ is a convex function satisfying $\max(t, t-1)\leq A(t) \leq 1.$ In the context of bivariate extreme value copulas, the functions $\ell$ and $A$ are related as follows:
+$$\ell(u_1,u_2)=(u_1+u_2)A\left(\frac{u_1}{u_1+u_2}\right).$$
+Therefore, in *TwinCopulas*, it is sufficient to provide the Pickands dependence function $A$ to construct the implementation structure of an extreme value copula.
```

Comment:
Package name ? 
Also, this whole technical explanation is very good, but it would be better with one or two bibliographical references

Reply:
Edit: I see that there are reference later on. But adding one or two here would be nice (could be the same ones)